packages/omg-js-util/src/index.js

```diff
@@ -18,5 +18,7 @@ module.exports = {
   transaction: require('./transaction'),
   sign: require('./sign'),
   hexToBytes: require('./hexToBytes'),
-  ArgTypes: require('./ArgTypes')
+  ArgTypes: require('./ArgTypes'),
+  utils: require('./utils')
```

Comment:
i think we should keep this specific and just export `hexPrefix` directly. because then its just `import { util } from omg-js-util` which is a generic util object exporting another generic util object

Reply:
👍